modules/installation-configuration-parameters.adoc

```diff
@@ -677,6 +677,10 @@ ifdef::gcp[If you are installing on GCP into a shared virtual private cloud (VPC
 ====
 Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the _Cloud Credential Operator_ entry in the _Cluster Operators reference_ content.
 ====
+[NOTE]
+====
+If your AWS account have SCP (Service control policies) enabled, you must modify `install-config.yaml` file and add the `credentialsMode: Mint` entry.
```

Comment:
@xenolinux all of `Mint`, `Passthrough` and `Manual` work with this situation.

cc @jianping-shu @huangmingxia for review.
cc @patrickdillon to be aware of this change.

Reply:
@yunjiang29  If the AWS account has SCP enabled, then do we have to enable all of these 3 entries? or do we only have to enable the `credentialsMode: Mint` entry?

Reply:
@xenolinux  only one mode can be specified in install-config: `Mint` or `Passthrough` or `Manual`

Reply:
Addressed